Dockerfile.envoy

```diff
@@ -27,7 +27,7 @@ ENV ICP_ENV_DIR /quickassist/build_system/build_files/env_files
 ENV ICP_BUILDSYSTEM_PATH /quickassist/build_system
 ENV KERNEL_SOURCE_ROOT /tmp
 
-RUN  ./configure --enable-qat-uio
+RUN  ./configure --disable-qat-lkcf
```

Comment:
Why is this option needed?

Reply:
With this option there's no need to deal with `[KERNEL_QAT]` sections in config files.
And `--enable-qat-uio` was removed from the `./configure` script.

Reply:
IMHO, Shouldn't be better to document the supported versions and leave this more *generic*? There is no guarantee that this tarball is that specific version, it only relies on the existence of `qat1.7.l.4.5.0-00034.tar.gz`

I suggest to change to:

```
ADD qat.tar.gz .
```

And modify the instruction in the README.md

Reply:
or how about ```ARG QAT_DRIVER_RELEASE="qat1.7.l.4.3.0-00033"``` which you can override using build args and then something like ```ADD $QAT_DRIVER_RELEASE.tar.gz```?

Reply:
This PR also updates the reference to device plugins' repo which is configured to work with qat1.7.l.4.5.0-00034.tar.gz too. If we make configuration more flexible it'll put additional cognitive load on end users.

Then the QAT team is creative enough to put QAT releases to different URL paths, so that configuring the file name alone is not enough. And since the DP loads the tarball automatically when building docker images we'll need to complicate the code.

I feel we need to unify the way how `Dockerfile.envoy` and the corresponding `Dockerfile` in DP get the tarball. But I'd do it in a separate PR.